client/rest/README_LIGHT.md

```diff
@@ -0,0 +1,22 @@
+# Light-rest
+
+## Introduction
+
+This is a lightweight version of the REST server. It provides a limited set of endpoints compared to the full API node. The main differences between the light rest and the full API node are:
+
+- The light rest does not use MongoDB or broker services.
+- it communicates with the peer node using a socket connection.
+
+## Purpose
+
+The purpose of the light rest is to reduce server resource usage, making it a cost-effective. it also offers users the ability to delegate harvesting to the peer node.
+
```

Comment:
Does Wallet still work? 🤔

Reply:
technically it should work, but I found out our statistic service didn't pick up `nodePublicKey`, something I need to fix and test it again in desktop wallet.

Reply:
I think the functionality available in the wallet will be limited if database and websocket is not available. 🤔
I will that for your testing.